{"uuid": "835ddcaf-449a-46dc-886f-4b3fc55c3eaa", "children": ["bab8cf41-955f-4016-9682-5a5123534cd7"], "befores": [{"name": "update_food_init", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range\n", "trace": "  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\DeliverySystem完整代码\\testCase\\conftest.py\", line 72, in update_food_init\n    idMenu = food_init.query({\"page\": 1, \"limit\": 1})['data']['records'][0]['category_id']\n"}, "start": 1704606285342, "stop": 1704606285523}], "start": 1704606285342, "stop": 1704606285538}